CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+> Note: This changelog only includes the changes for the stable versions of VSCode API Center. For the changelog of pre-released versions, please refer to the [VSCode APICenter Changelog](https://github.com/microsoft/vscode-azureapicenter/blob/main/PRERELEASE.md).
```

Comment:
'VSCode API Center' -> 'Azure API Center for Visual Studio Code'
[VSCode APICenter Changelog] -> [Pre-release Changelog of Azure API Center for Visual Studio Code]

Reply:
updated.